specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -440,6 +440,99 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BatchAI/locations/{location}/quotas": {
```

Comment:
POST apis should start with a verb, i.e. `updateQuotas`

Reply:
Thanks for the feedback, I am really appreciated it! I will modify those and update to preview.